hooks/vars.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
 
 export VARS='
+TAG="latest" DOCKER_CONTEXT="debian" BASE_IMAGE_TAG="11.0.4-jdk-stretch" SBT_VERSION="1.3.3" SCALA_VERSION="2.13.1"
```

Comment:
Please remove.

Reply:
was mainly to also tackle  #93 but if you don't want that you should probably just close that ticket